### DISCUSSION TITLE
score_ not available in WeibullAFTFitter

### DISCUSSION THREAD

**Discussion by shj997** [CamDavidsonPilon/lifelines#1380] [Q&A] [answered]
I am trying to create a feature importance function for a Weibull model but get an error saying 'WeibullAFTFitter' object has no attribute 'score_'.  Please assist.

**Comment by CamDavidsonPilon** [comment, marked as answer]
I think my documentation is out-of-date. Try `.concordance_index_` instead, or `.score(df)`
